dockerfiles/mssql2025-container-disk:c10s/install_mssql.sh

```
@@ -0,0 +1,33 @@
+
+#In case install Selinux: - need to disable it
+$ vi /etc/selinux/config [Selinux turned off - Solved Permission denied]
+SELINUX=disabled
+
+$ sestatus [after reboot]
+SELinux status: disabled
+
+#Install odbc18:
+Verify:
+cat /etc/odbcinst.ini OR odbcinst -q -d
+[ODBC Driver 18 for SQL Server]
```

Shellcheck (0.11.0)

[error] You need a space after the [ and before the ].

(SC1035)